### DISCUSSION TITLE
Uint8Array of enums does not display enum name in State Transition Panel

### DISCUSSION THREAD

**Discussion by jameskuszmaul-brt** [foxglove/community#978] [Bugs]
![image](https://github.com/foxglove/community/assets/91565539/e967b404-5445-43f2-96d9-cb3395f67302)

If I have a message with enum types that are backed by `uint8`s and that enum is in a vector then I can't plot `/channel.enum_vector[0]` in the State Transition panel and have it give me the enum name; instead it gives me the numeric value of the enum. It does display correctly in the Raw Message panel.

**Comment by defunctzombie** [comment]
Please provide a sample mcap file to show the problem. The mcap file is necessary so we can see what format you are using for message data and how enums are specified.

**Comment by defunctzombie** [comment]
Closing since we've not gotten a sample file to investigate. Happy to revisit if a sample is provided (you know where to find us :D)
